src/js/stores/AppsStore.js

```diff
@@ -295,7 +314,7 @@ QueueStore.on(QueueEvents.CHANGE, function () {
 AppDispatcher.register(function (action) {
   switch (action.actionType) {
     case AppsEvents.REQUEST_APPS:
-      storeData.apps = processAppsAndEmptyGroups(action.data.body);
+      storeData.apps = processAppsAndGroups(action.data.body);
```

Comment:
No Orlando is right the apss array will contain all apps and groups in one flat list. which is called apps in the function which was a bad decision.